packages/shared/src/features/briefing/components/BriefFeedBanner.tsx

```diff
@@ -0,0 +1,103 @@
+import type { ComponentProps } from 'react';
+import React, { useCallback, useEffect, useRef } from 'react';
+import { useRouter } from 'next/router';
+import classNames from 'classnames';
+import {
+  Typography,
+  TypographyColor,
+  TypographyType,
+} from '../../../components/typography/Typography';
+import {
+  Button,
+  ButtonSize,
+  ButtonVariant,
+} from '../../../components/buttons/Button';
+import { DevPlusIcon, TimerIcon } from '../../../components/icons';
+import { briefButtonBg } from '../../../styles/custom';
+import { LogEvent, TargetId } from '../../../lib/log';
+import { useLogContext } from '../../../contexts/LogContext';
+import { useAuthContext } from '../../../contexts/AuthContext';
+
+export const BriefFeedBanner = (props: ComponentProps<'div'>) => {
+  const router = useRouter();
+  const { className, style, ...attrs } = props;
+
+  const impressionRef = useRef(false);
+  const { logEvent } = useLogContext();
+  const { user, isAuthReady } = useAuthContext();
+
+  const logBriefEvent = useCallback(
+    (eventName: LogEvent) => {
+      logEvent({
+        event_name: eventName,
+        target_id: TargetId.BriefPlacement,
+        extra: JSON.stringify({
+          is_demo: !user?.isPlus,
+          brief_date: new Date(),
+        }),
+      });
+    },
+    [logEvent, user?.isPlus],
+  );
+
+  useEffect(() => {
+    if (impressionRef.current || !isAuthReady) {
+      return;
+    }
+
+    impressionRef.current = true;
+    logBriefEvent(LogEvent.ImpressionBrief);
+  }, [isAuthReady, logBriefEvent]);
+
+  return (
+    <div
```

Comment:
Not sure if the colors work well in light mode?
Struggle to read this

<img width="1145" height="268" alt="Image" src="https://github.com/user-attachments/assets/bb9509e5-c532-4a5e-9f3e-6ad9bb7fc228" />